lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,21 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str) or len(text) < 1:
+        return None
+
+    tokens = ''
+    word = ''
+    for symbol in text.lower():
+        if symbol.isalpha() or symbol == ' ':
```

Comment:
use continue if this condition is not True

Reply:
Done (changed the function and used continue in the similar situation)